src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -2649,6 +2649,9 @@ static bool TakeOutParam(object y, out bool x)
                 // (98,29): error CS0103: The name 'y12' does not exist in the current context
                 //         while (TakeOutParam(y12, out var x12));
                 Diagnostic(ErrorCode.ERR_NameNotInContext, "y12").WithArguments("y12").WithLocation(98, 29),
+                // (97,17): warning CS0219: The variable 'y12' is assigned but its value is never used
+                //             var y12 = 12;
+                Diagnostic(ErrorCode.WRN_UnreferencedVarAssg, "y12").WithArguments("y12").WithLocation(97, 17),
```

Comment:
Oh, I see it is preceded by an error for an embedded declaration. Good.

---
In reply to: [75559057](https://github.com/dotnet/roslyn/pull/13271#discussion_r75559057) [](ancestors = 75559057)